tensorflow_lite_support/python/test/task/text/text_embedder_test.py

```diff
@@ -105,14 +105,21 @@ def test_embed(self, model_name, l2_normalize, quantize, model_file_type,
     def _check_embedding_size(result):
       self.assertLen(result.embeddings, 1)
       feature_vector = result.embeddings[0].feature_vector
-      if quantize:
-        self.assertLen(feature_vector.value_string, embedding_length)
-      else:
-        self.assertLen(feature_vector.value_float, embedding_length)
+      self.assertLen(feature_vector.value, embedding_length)
```

Comment:
There's no need to define helper function like this. You can write each assert in one line just like in the audio_embedder_test.py

Reply:
@khanhlvg I've simplified the tests now.